### DISCUSSION TITLE
Homescreen wallpaper slightly offset to the right?

### DISCUSSION THREAD

**Discussion by groggone** [FossifyOrg/General-Discussion#405] [Q&A] [answered]
Is it an intentional design that if an image is set as a wallpaper using the built-in wallpaper setter, on the homescreen it's slightly offset to the right meanwhile on the lockscreen it's not? If it is, is there a way to change this? To not do the slight offset to the right?

**Comment by Aga-C** [comment, marked as answer]
The home screen may have a little offset because launchers move wallpaper a bit to the left or right while switching screens. On the other hand, the lock screen is only one, so it's just centered or moved to the left.
